src/components/dev-hub/tab.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import { colorMap, size, fontSize } from './theme';
+import { P } from '../dev-hub/text';
+
+const Tab = styled('div')`
+    display: flex;
+    justify-content: space-between;
+    border-bottom: 1px solid ${colorMap.greyDarkOne};
+`;
+
+const activeStyles = css`
+    color: ${colorMap.devWhite};
+    border-bottom-color: ${colorMap.darkGreen};
+`;
+
+const defaultStyles = css`
+    &:hover {
+        color: ${colorMap.devWhite};
+    }
+`;
+
+const TabButton = styled('button')`
+    border: none;
+    outline: none;
+    cursor: pointer;
+    transition: 0.3s;
+    background-color: inherit;
+    width: 136px;
+    padding-top: ${size.micro};
```

Comment:
Nitpicky, but can we reduce this to [one-line padding css](https://www.w3schools.com/css/css_padding.asp): `padding: ${size.micro} ${size.medium} ${size.mediumLarge} ${size.mediumLarge}`